arch/risc-v/src/mpfs/Kconfig

```diff
@@ -416,8 +416,6 @@ config MPFS_EMMCSD
 	select ARCH_HAVE_SDIO
 	select SDIO_BLOCKSETUP
 	select SDIO_DMA
-	select FAT_DMAMEMORY
```

Comment:
Does this work with kernel mode? If I remember correctly we had to enable FORCE_INDIRECT and for that we needed FAT_DMAMEMORY, because otherwise the MMC driver and/or MMC DMA would write directly to user space memory. DMA only works with physical addresses (and user addresses are virtual) so this fails miserably.

FORCE_INDIRECT makes the file system read the media in 512B blocks, which are properly aligned AND DMA safe! These 512B blocks are then moved to the user buffer.

Reply:
Yes, this does not work in kernel mode as such. So, change not usable.